tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,374 @@
-# Compute inverse solution during the first 15s:
+###############################################################################
+# We can also get a more detailed view of each :class:`~mne.Evoked` object
+# using other plotting methods such as :meth:`~mne.Evoked.plot_joint` or
+# :meth:`~mne.Evoked.plot_topomap`. Here we'll examine just the EEG channels,
+# and see the classic auditory evoked N100-P200 pattern over dorso-frontal
+# electrodes, then plot scalp topographies at some additional arbitrary times:
 
-from mne.minimum_norm import apply_inverse_raw  # noqa
-start, stop = raw.time_as_index([0, 15])  # read the first 15s of data
-stc = apply_inverse_raw(raw, inverse_operator, lambda2, method, label,
-                        start, stop)
+# sphinx_gallery_thumbnail_number = 13
+aud_evoked.plot_joint(picks='eeg')
+aud_evoked.plot_topomap(times=[0., 0.08, 0.1, 0.12, 0.2], ch_type='eeg')
 
 ##############################################################################
-# Save result in stc files:
+# Evoked objects can also be combined to show contrasts between conditions,
+# using the :func:`mne.combine_evoked` function. A simple difference can be
+# generated by negating one of the :class:`~mne.Evoked` objects passed into the
+# function. We'll then plot the difference wave at each sensor using
+# :meth:`~mne.Evoked.plot_topo`:
 
-stc.save('mne_dSPM_raw_inverse_Aud')
+evoked_diff = mne.combine_evoked([aud_evoked, -vis_evoked], weights='equal')
+evoked_diff.pick_types('mag').plot_topo(color='r', legend=False)
 
 ##############################################################################
-# What else can you do?
-# ^^^^^^^^^^^^^^^^^^^^^
+# Inverse modeling
+# ^^^^^^^^^^^^^^^^
 #
-#     - detect heart beat QRS component
-#     - detect eye blinks and EOG artifacts
-#     - compute SSP projections to remove ECG or EOG artifacts
-#     - compute Independent Component Analysis (ICA) to remove artifacts or
-#       select latent sources
-#     - estimate noise covariance matrix from Raw and Epochs
-#     - visualize cross-trial response dynamics using epochs images
-#     - compute forward solutions
-#     - estimate power in the source space
-#     - estimate connectivity in sensor and source space
-#     - morph stc from one brain to another for group studies
-#     - compute mass univariate statistics base on custom contrasts
-#     - visualize source estimates
-#     - export raw, epochs, and evoked data to other python data analysis
-#       libraries e.g. pandas
-#     - and many more things ...
+# Finally, we can estimate the origins of the evoked activity by projecting the
+# sensor data into this subject's :term:`source space` (a set of points either
+# on the cortical surface or within the cortical volume of that subject, as
+# estimated by structural MRI scans). MNE-Python supports lots of ways of doing
+# this (dynamic statistical parametric mapping, dipole fitting, beamformers,
```

Comment:
this ways should point to their references (and list here all the most common)

Reply:
I'm skipping this because I think there isn't an obvious single place for each of these terms to link to.  Which means they ought to have glossary entries probably.  I'll open a "enhance the glossary" issue where we can keep track of these things.

Reply:
glossary TODO is #6321